django/thunderstore/api/urls.py

```diff
@@ -18,6 +19,11 @@
 )
 
 cyberstorm_urls = [
+    path(
+        "dynamichtml/<str:placement>/",
+        DynamicHTMLAPIView.as_view(),
+        name="cyberstorm.dynamichtml",
+    ),
```

Comment:
Perhaps we should have an API view which returns all active placements, rather than one view per placement ID? Would make it only require 1 request on the clients